src/install/troubleshooting.rst

```diff
@@ -275,3 +275,15 @@ the relevant CouchDB and then compact prior to replicating.
 
 Alternatively, if the number of documents impacted is small, use filtered
 replication to exclude only those documents.
+
+Mac Os Known Issues
```

Comment:
Wonder if Mac OS or macOS would be the more correct name.

Reply:
Oh yes, it’s named macOS before